packages/grafana-ui/src/components/TimePicker/_TimePicker.scss

```diff
@@ -250,6 +251,24 @@ $arrowPadding: $arrowPaddingToBorder * 3;
       }
     }
   }
+
+  .time-picker-long-date {
+    .select-button {
+      span {
```

Comment:
now we only have this internal override on an internal span element that belongs to the ButtonSelect component. So this time-picker-long-date class should be a property or BEM modifier class on the ButtonSelect not on the TimePicker component.